src/paths/distances.c

```diff
@@ -161,7 +186,8 @@ int igraph_eccentricity(const igraph_t *graph,
                         igraph_vs_t vids,
                         igraph_neimode_t mode) {
 
-    return igraph_i_eccentricity(graph, res, vids, mode, /*adjlist=*/ 0);
+    return igraph_i_eccentricity(graph, res, vids, mode, /*adjlist=*/ 0,
```

Comment:
If we agreed that we will spell out null pointers explicitly with `NULL`, let's change it for `adjlist=...` as well for sake of consistency.